File: Examples/src/main/java/com/aspose/pdf/examples/AsposePdf/Pages/GetNumberofPages.java
package com.aspose.pdf.examples.AsposePdf.Pages;

import com.aspose.pdf.examples.Utils;

public class GetNumberofPages
{
    public static void main(String[] args) throws Exception
    {
        // The path to the documents directory.
        String dataDir = Utils.getDataDir(GetNumberofPages.class);

        //open document
        com.aspose.pdf.Document pdfDocument = new com.aspose.pdf.Document(dataDir + "input1.pdf");

        //get page count
        System.out.printf("Page Count :- " + pdfDocument.getPages().size());

    }
}

File: Plugins/Aspose-Pdf-Java-for-Jython/asposepdf/WorkingWithPages/GetNumberOfPages.py
from asposepdf import Settings
from com.aspose.pdf import Document
from java.util import Date


class GetNumberOfPages:

    def __init__(self):
        
        dataDir = Settings.dataDir + 'WorkingWithPages/GetNumberOfPages/'
        
        # Create PDF document
        pdf = Document(dataDir + 'input1.pdf')

        page_count = pdf.getPages().size()

        print "Page Count:"
        print page_count
 
if __name__ == '__main__':        
    GetNumberOfPages()